router/router.go

```diff
@@ -11,14 +11,16 @@ import (
 func StartApp(db *sql.DB) *gin.Engine {
 	router := gin.Default()
 
-	router.Use(func(c *gin.Context) {
-		c.Set("db", db)
-		c.Next()
+	userRouter := router.Group("v1/user")
+	{
+		userRouter.POST("/register", controllers.UserRegister)
```

Comment:
Did payload validation works without intercepted by middleware? Try case with name more than 15 and password more than 15